pkg/registry/servicecatalog/instance/strategy.go

```diff
@@ -123,6 +123,8 @@ func (instanceRESTStrategy) PrepareForUpdate(ctx genericapirequest.Context, new,
 	}
 
 	// TODO: ensure that checksum is carried forward when we allow updates to the spec.
+	// TODO(vaikas): This seems like it would mean an object can't be updated, but seems
```

Comment:
Yeah, I found it, but I don't understand if there are cases where we use one or the other, or if we only ever use the InstanceStatusRESTStrategy.PrepareUpdate?
If there are cases, where do we dispatch to appropriate PrepareUpdate strategy
If there are not, why do we have two methods.